### PR TITLE
i18n(zh-cn): translate reference/image-service-reference.mdx

### PR DESCRIPTION
**Pull request by Lilian97**
#### What kind of changes does this PR include?

- Translated content

#### Description

- translate reference/image-service-reference.mdx

### CODE REVIEW

File: src/content/docs/zh-cn/reference/image-service-reference.mdx [hunk 1]
```diff
@@ -0,0 +1,241 @@
+---
+title: 图像服务 API
+i18nReady: true
+---
+
+Astro [assets]（/en/guides/assets/） 旨在使任何图像优化服务都能轻松地在 Astro 上构建服务。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/image-service-reference.mdx:6]:
```suggestion
Astro [assets](/zh-cn/guides/assets/) 旨在使任何图像优化服务都能轻松地在 Astro 上构建服务。
```

File: src/content/docs/zh-cn/reference/image-service-reference.mdx [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+
+:::caution
+图像服务 API 是 Astro v2.1 中引入的一个实验性功能。该 API 在标记为稳定之前可能会发生更改。
+:::
+
+## 什么是图像服务？
+
+Astro 提供两种类型的图像服务：本地和外部。
+
+- **本地服务**在构建静态站点时直接处理图像转换，或者在运行时同时处理开发模式和 SSR 中的图像转换。这些通常是围绕 Sharp、ImageMagick 或 Squoosh 等库的包装。在开发模式和 SSR 中，本地服务使用 API 端点进行转换。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/image-service-reference.mdx:16]:
```suggestion
- **本地服务**在构建静态站点时直接处理图像转换，或者在运行时同时处理开发模式和 SSR 中的图像转换。这些通常是围绕 Sharp、ImageMagick 或 Squoosh 等库的封装。在开发模式和 SSR 中，本地服务使用 API 端点进行转换。
```

File: src/content/docs/zh-cn/reference/image-service-reference.mdx [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+- **外部服务**指向 URL，可以添加对 Cloudinary、Vercel 或任何符合 [RIAPI](https://github.com/riapi/riapi) 的服务器等服务的支持。
+
+## 使用图像服务 API 构建
+
+服务定义采用导出默认对象的形式，具有各种必需的方法（“钩子”）。
+
+外部服务提供一个 `getURL()`， 它指向输出 `＜img＞` 标记的 `src`。
+
+本地服务为开发模式和 SSR 提供 `transform()` 方法来对图像执行转换，并提供 `getURL()` 和 `parseURL()` 方法来使用端点。
+Local services provide a `transform()` method to perform transformations on your image, and  `getURL()` and `parseURL()` methods to use an endpoint for dev mode and SSR.
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/image-service-reference.mdx:26]: 这里怎么还有英文

File: src/content/docs/zh-cn/reference/image-service-reference.mdx [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+**本地和外部服务所需**
+
+`getURL(options: ImageTransform, imageServiceConfig: Record<string, any>): string`
+
+对于本地服务，该钩子返回生成图像的端点的 URL（在 SSR 和开发模式下）。 在构建期间它不可用。 `getURL()` 指向的本地端点可以调用 `parseURL()` 和 `transform()`。
+
+对于外部服务，该钩子返回图像的最终 URL。
+
+对于这两种类型的服务，`options`是用户作为 `<Image />` 组件的属性或 `getImage()` 选项传递的属性。 它们属于以下类型：
+For both types of services, `options` are the properties passed by the user as attributes of the `<Image />` component or as options to `getImage()`. They are of the following type:
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/image-service-reference.mdx:169]: 英文原文

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**